packages/create-fuels/src/lib/setupProgram.ts

```diff
@@ -22,6 +26,7 @@ export const setupProgram = () => {
     .option('--bun', 'Use bun to install dependencies')
     .option('--verbose', 'Enable verbose logging')
     .option('--no-install', 'Do not install dependencies')
+    .option('--template <template>', 'Specify a template to use')
```

Comment:
Could we add a `defaultValue` here?

```suggestion
    .option('--template <template>', 'Specify a template to use', 'nextjs')
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2851/commits/4f1720a14e2b16bc59b6d9c14f634e8d4aade15b